### PR TITLE
Add Polish localization - pl.toml

### PR DESCRIPTION
**Pull request by mfilocha**
This is a part of the initial Polish localization - see: https://github.com/kubernetes/website/pull/18419

### CODE REVIEW

File: i18n/pl.toml [hunk 1]
```diff
@@ -0,0 +1,191 @@
+# i18n strings for the Polish site.
+# NOTE: Please keep the entries in alphabetical order when editing
+
+[caution]
+other = "Ostrzeżenie:"
+
+[cleanup_heading]
+other = "Cleaning up"
```
Comment (sftim) [inline i18n/pl.toml]: nit: not yet localized

File: i18n/pl.toml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+other = """Nasz projekt jest uznany przez <a href="https://cncf.io/">CNCF</a> za dojrzały</p>"""
+
+[main_community_explore]
+other = "Poznaj społeczność"
+
+[main_contribute]
+other = "Wnieś swój wkład"
+
+[main_copyright_notice]
+other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
```
Comment (sftim) [inline i18n/pl.toml]: nit: not yet localized

File: i18n/pl.toml [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+[main_documentation_license]
+other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
```
Comment (sftim) [inline i18n/pl.toml]: nit: not yet localized

File: i18n/pl.toml (hunk 2 above)
Comment (mfilocha) [reply i18n/pl.toml]: I'm not going to translate this copyright notice.

File: i18n/pl.toml (hunk 3 above)
Comment (mfilocha) [reply i18n/pl.toml]: I'm not going to translate this licence notice.

File: i18n/pl.toml (hunk 2 above)
Comment (sftim) [reply i18n/pl.toml]: There are certainly localizations that skip translating it. For a counter-example, see https://kubernetes.io/zh/

File: i18n/pl.toml (hunk 1 above)
Comment (mfilocha) [reply i18n/pl.toml]: Fixed.

File: i18n/pl.toml [hunk 4]
```diff
@@ -0,0 +1,191 @@
+# i18n strings for the Polish site.
+# NOTE: Please keep the entries in alphabetical order when editing
+
+[caution]
+other = "Ostrzeżenie:"
+
+[cleanup_heading]
+other = "Porządkujemy po sobie"
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Sprzątamy po sobie"
```

File: i18n/pl.toml [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+other = "Stack Overflow"
+
+[community_twitter_name]
+other = "Twitter"
+
+[community_youtube_name]
+other = "YouTube"
+
+[deprecation_warning]
+other = " dokumentacja nie jest juz aktualizowana. Wyświetlona jest wersja archiwalna. Po aktualną dokumentację zajrzyj na"
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = " dokumentacja nie jest już aktualizowana. Wyświetlona jest wersja archiwalna. Po aktualną dokumentację zajrzyj na"
```

File: i18n/pl.toml [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+other = "<< Poprzedni"
+
+[layouts_blog_pager_next]
+other = "Następny >>"
+
+[layouts_case_studies_list_tell]
+other = "Opowiedz swoją historię"
+
+[layouts_docs_glossary_aka]
+other = "Znany tez jako"
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Znany też jako"
```

File: i18n/pl.toml [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+[layouts_docs_glossary_description]
+other = "Celem tego słownika jest przedstawienie wszechstronnej, ujednoliconej listy terminologii związanej z Kubernetes. Słownik zawiera terminy specyficzne dla Kubernetes, a także pojęcia bardziej ogólne, dające lepsze zrozumienie kontekstu."
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Celem tego słownika jest przedstawienie wszechstronnej, ujednoliconej listy terminologii związanej z projektem Kubernetes. Słownik zawiera terminy specyficzne dla Kubernetesa, a także pojęcia bardziej ogólne, umożliwiające lepsze zrozumienie kontekstu."
```

File: i18n/pl.toml [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+other = "Otwórz zgłoszenie w repozytorium GitHub, jeśli chcesz "
+
+[layouts_docs_partials_feedback_or]
+other = "lub"
+
+[layouts_docs_partials_feedback_problem]
+other = "zgłosić problem"
+
+[layouts_docs_partials_feedback_thanks]
+other = "Dziękujemy za informację zwrotną. Jeśli masz konkretne pytanie dotyczące użycia Kubernetes, odwiedź"
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Dziękujemy za informację zwrotną. Jeśli masz konkretne pytanie dotyczące użycia Kubernetesa, odwiedź"
```

File: i18n/pl.toml [hunk 9]
```diff
@@ -0,0 +1,191 @@
...
+other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
+
+[main_edit_this_page]
+other = "Edytuj stronę"
+
+[main_github_create_an_issue]
+other = "Zgłoś problem"
+
+[main_github_invite]
+other = "Chcesz zacząć współtworzyć kod Kubernetes?"
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Chcesz zacząć współtworzyć kod Kubernetesa?"
```

File: i18n/pl.toml [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+
+[main_github_view_on]
+other = "Zajrzyj na GitHub"
+
+[main_kubernetes_features]
+other = "Funkcjonalności Kubernetes"
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Funkcjonalności Kubernetesa"
```

File: i18n/pl.toml [hunk 11]
```diff
@@ -0,0 +1,191 @@
...
+
+[main_kubeweekly_baseline]
+other = "Zapisz się na KubeWeekly, jeśli jesteś zainteresowany najnowszymi wiadomościami o Kubernetes."
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Zapisz się na KubeWeekly, jeśli jesteś zainteresowany najnowszymi wiadomościami o Kubernetesie."
```

File: i18n/pl.toml [hunk 12]
```diff
@@ -0,0 +1,191 @@
...
+other = "Cele"
+
+[prerequisites_heading]
+other = "Nim zaczniesz"
+
+[ui_search_placeholder]
+other = "Szukaj"
+
+[version_check_mustbe]
+other = "Twój serwer Kubernetes musi być w wersji "
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Twój serwer Kubernetesa musi być w wersji "
```

File: i18n/pl.toml [hunk 13]
```diff
@@ -0,0 +1,191 @@
...
+
+[version_check_mustbeorlater]
+other = "Twój serwer Kubernetes musi być co najmniej w wersji "
```
Comment (kpucynski) [inline i18n/pl.toml]:
```suggestion
other = "Twój serwer Kubernetesa musi być co najmniej w wersji "
```

File: i18n/pl.toml (hunk 12 above)
Comment (mfilocha) [reply i18n/pl.toml]: I'm not sure if this one should inflect.

File: i18n/pl.toml (hunk 13 above)
Comment (mfilocha) [reply i18n/pl.toml]: I'm not sure if this one should inflect.

**Comment by sftim** [comment]
/hold
pending approval & merge for #18419. Once that merges, it's OK to /unhold
People can /lgtm (for localization review) and /approve, even whilst this is held.

File: i18n/pl.toml (hunk 13 above)
Comment (kpucynski) [reply i18n/pl.toml]: fine for me

File: i18n/pl.toml (hunk 12 above)
Comment (kpucynski) [reply i18n/pl.toml]: fine for me

**Comment by rlenferink** [comment]
_Approved based on `/lgtm` and `/approve` of @mfilocha and @nvtkaszpir_

**Pull request merged**